libcontainer/factory_linux.go

```diff
@@ -196,7 +196,7 @@ func New(root string, options ...func(*LinuxFactory) error) (Factory, error) {
 		Validator: validate.New(),
 		CriuPath:  "criu",
 	}
-	Cgroupfs(l)
+	_ = Cgroupfs(l)
```

Comment:
Maybe it's better to return an error from here rather than ignore it.

Reply:
updated (not pushed yet)